samples/dnn/person_reid.cpp

```diff
@@ -337,7 +301,7 @@ static void drawRectangle(int event, int x, int y, int, void* param) {
     }
 }
 
-void extractFrames(Net* reidNet) {
+void extractFrames(Net& reidNet) {
```

Comment:
The dedicated function does not make a lot of sense. I propose to merge it inside of main.